shared-libs/transitions/test/unit/transitions/update_clinics.js

```diff
@@ -300,20 +300,84 @@ describe('update clinic', () => {
     };
 
     sinon.stub(db.medic, 'query').resolves({ rows: [{ key: '123' }] });
-    sinon.stub(config, 'get').withArgs('forms').returns({ 'someForm': {} });
+    const stubbedConfig = sinon.stub(config, 'get');
+    stubbedConfig.returns([ {
+      form: 'someForm',
+      messages: [
+        {
+          event_type: 'sys.facility_not_found',
+          recipient: 'reporting_unit',
+          translation_key: 'sys.facility_not_found',
+        }
+      ],
+    }]);
+    stubbedConfig.withArgs('forms').returns({ 'someForm': {} });
     sinon.stub(utils, 'translate').returns('translated');
     sinon.stub(utils, 'getLocale').returns('locale');
 
     return transition.onMatch({ doc }).then(changed => {
       assert(changed);
       assert(!doc.contact);
       assert.equal(doc.errors.length, 1);
-      assert.deepEqual(doc.errors[0], { code: 'sys.facility_not_found', message: 'translated' });
-      assert.equal(utils.translate.callCount, 1);
+      assert.deepEqual(doc.errors[0], {
+        code: 'sys.facility_not_found',
+        message: 'translated'
+      });
+      assert.equal(utils.translate.callCount, 2); // called by messages.addMessage and messages.getMessage
```

Comment:
I think we should add an assertion here for `doc.tasks`, and also about the two calls to `utils.translate`